db/seeds.rb

```diff
@@ -0,0 +1,4 @@
+ReviewMetric.create!(metric: 'Volume')
```

Comment:
You may add the content to [factories.rb](https://github.com/expertiza/expertiza/blob/master/spec/factories/factories.rb).